auto_editor/AddCommentsRule.py

```diff
@@ -13,10 +13,51 @@ def change_type(self) -> ChangeTypeEnum:
     @property
     def dpct_warning_codes(self) -> List[str]:
         # Todo: add relevant warning codes
-        return ['DPCT1111']
+        return ['DPCT1111','DPCT1065']
 
     def run_rule(self, project: StructuredProjectSource,
                  warning_first_line: int, warning_last_line: int, file_path: str) -> StructuredProjectSource:
         # Todo: add rule here
-        print(f'rule runs here. warning first: {warning_first_line}')
+        # print(f'rule runs here. warning first: {warning_first_line}')
+        # return project
+
+        state = 0
+        w_type = "DPCT"
+        new_line_items = list()
+
+        all_line = list()
+        for l in project.get_line_items(file_path):
```

Comment:
as I said earlier, please don't use .get_line_items() here, but instead use .paths_to_lines. Otherwise it breaks the changes tracker, because it recreates all line items from scratch with new random id's.